client/src/app/pages/waves/waves.tsx

```diff
@@ -45,23 +46,24 @@ import {
   TableHeaderContentWithControls,
   TableRowContentWithControls,
 } from "@app/shared/components/table-controls";
+dayjs.extend(utc);
 
 export const Waves: React.FC = () => {
   const { t } = useTranslation();
 
-  const { waves, isFetching, fetchError } = useFetchWaves();
+  const { waves, isFetching, fetchError } = useFetchMigrationWaves();
 
   // When waveModalState is:
   //   'create': the modal is open for creating a new wave.
   //   A Wave: the modal is open for editing that wave.
   //   null: the modal is closed.
   const [waveModalState, setWaveModalState] = React.useState<
-    "create" | Wave | null
+    "create" | MigrationWave | null
```

Comment:
Above comment needs to be updated too or removed.
In my opinion those comments are redundant and not necessary as the const name is pretty explicit necessary but it's up to you.